frontend/src/components/db/CurrentResultChips.tsx

```diff
@@ -1,112 +1,106 @@
 import React from "react";
-import { useRouter } from "next/router";
+import { ReadonlyURLSearchParams, useSearchParams } from "next/navigation";
 import { useTranslation } from "next-i18next";
 import ParallelsChip from "@components/db/ParallelsChip";
 import SearchMatchesChip from "@components/db/SearchMatchesChip";
-import { useDbQueryParams } from "@components/hooks/useDbQueryParams";
 import {
-  customFiltersChipQueryExclusions,
-  customOptionsChipQueries,
-} from "@features/sidebarSuite/config";
-import { uniqueSettings } from "@features/sidebarSuite/config/settings";
-import type {
-  DefaultQueryParams,
-  LimitsFilterValue,
-  MultiLingalParam,
-  QueryParams,
-} from "@features/sidebarSuite/config/types";
+  ResultPageType,
+  useResultPageType,
+} from "@components/hooks/useResultPageType";
+import {
+  DBSourceFilePageFilterUISettingName,
+  DisplayUISettingName,
+  SearchPageFilterUISettingName,
+} from "@features/SidebarSuite/types";
+import {
+  dbSoureFileRequestFilters,
+  displayUISettings,
+  searchRequestFilters,
+} from "@features/SidebarSuite/uiSettings/config";
 import Chip from "@mui/material/Chip";
-import { useQuery } from "@tanstack/react-query";
-import { DbApi } from "@utils/api/dbApi";
-
-type ParamValues = string | number | LimitsFilterValue | MultiLingalParam;
-
-function getFilterCount(key: string) {
-  return customFiltersChipQueryExclusions.includes(key) ? 0 : 1;
-}
-function getDisplayOptionCount(key: string) {
-  return customOptionsChipQueries.includes(key) ? 1 : 0;
-}
 
-function isDefaultValue({
-  defaultQueries,
-  queryKey,
-  value,
-}: {
-  defaultQueries: DefaultQueryParams;
-  queryKey: keyof typeof defaultQueries;
-  value: ParamValues;
-}) {
-  // `defaultQueries` correspond to params that must be set for API calls. The `defaults` variable here defines the default settings used in API results, but aren't necessarily set on the FE (eg. `multi_lingual`)
-  // TODO: handling here is being reviewed for refactoring and this may be removable.
-  const defaults = {
-    ...defaultQueries,
-    [uniqueSettings.queryParams.sortMethod]: "position",
-    [uniqueSettings.queryParams.folio]: null,
-  };
-  return defaults[queryKey]?.toString() === value;
-}
+const searchFilterSet = new Set(searchRequestFilters);
+const dbSourceFileFilterSet = new Set(dbSoureFileRequestFilters);
+const displayParamSet = new Set(displayUISettings);
```

Comment:
nitpick: I would suggest using the same name for both (choose either displayParam vs displayUISettings)